ouroboros-network-framework/test/Test/Ouroboros/Network/Socket.hs

```diff
@@ -0,0 +1,361 @@
+{-# LANGUAGE BangPatterns        #-}
```

Comment:
`git mv`

Reply:
This is moving a part of the code from `ouroboros-network/test/Test/Socket.hs`, but not all of it. The original file remains, with the tests that still belong in the `ouroboros-network` package. So that's why it cannot be a git mv.